lib/internal/util.js

```diff
@@ -18,7 +18,11 @@ exports.printDeprecationMessage = function(msg, warned) {
 };
 
 exports.error = function(msg) {
-  console.error(`${prefix}${msg}`);
+  if (arguments.length > 1) {
+    arguments[0] = `${prefix}${msg}`;
```

Comment:
It probably doesn't really matter here but V8 doesn't like it when you modify `arguments`.